src/NHibernate/Driver/IDriver.cs

```diff
@@ -143,6 +143,12 @@ public interface IDriver
 		/// </summary>
 		bool SupportsEnlistmentWhenAutoEnlistmentIsDisabled { get; }
 
+		/// <summary>
+		/// Does this driver connections support being closed during a scope then re-acquired during prepare, including
+		/// in case of concurrent rollback?
+		/// </summary>
+		bool SupportsClosingConnectionInsideScopeWithEnlistDuringPrepare { get; }
```

Comment:
Interface change....

Reply:
Moreover... Anyway, since even the one responsible for how MSDTC+.Net works together, does not get it right when we close a connection before completion then re-open one in prepare phase, better forget about this.